rclpy/CMakeLists.txt

```diff
@@ -14,8 +14,10 @@ find_package(PythonExtra MODULE)
 set(_PYTHON_EXECUTABLE "${PYTHON_EXECUTABLE}")
 if(NOT WIN32)
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra")
-elseif(CMAKE_BUILD_TYPE STREQUAL "Debug")
-  set(PYTHON_EXECUTABLE "${PYTHON_EXECUTABLE_DEBUG}")
+else(NOT WIN32)
+  if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+    set(PYTHON_EXECUTABLE "${PYTHON_EXECUTABLE_DEBUG}")
+  endif()
```

Comment:
Sorry for the noise, this should still be in progress, I'm still investigating why the python executable used is not the one expected a test time